resilience4j-bulkhead/src/main/java/io/github/resilience4j/bulkhead/adaptive/LimitResult.java

```diff
@@ -0,0 +1,50 @@
+package io.github.resilience4j.bulkhead.adaptive;
+
+import java.util.Objects;
+
+/**
+ * limit result DTO
+ */
+public class LimitResult {
```

Comment:
Could you also add some documentation about the usage of LimitResult. What does the AdaptiveBulkhead do with this LimitResult. It should help developers to implement new policies.